pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -120,6 +120,7 @@ func init() {
 		Rules: []rbac.PolicyRule{
 			rbac.NewRule("get", "list", "watch").Groups(buildGroup, legacyBuildGroup).Resources("buildconfigs").RuleOrDie(),
 			rbac.NewRule("create").Groups(buildGroup, legacyBuildGroup).Resources("buildconfigs/instantiate").RuleOrDie(),
+			rbac.NewRule("delete").Groups(buildGroup, legacyBuildGroup).Resources("builds").RuleOrDie(),
```

Comment:
why doesn't the build-controller also need this permission?  doesn't it delete builds also?

Reply:
nm, i'm blindish.